scripts/run_app.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 
 yarn install
+DB_CONNECTION_URI=postgres://postgres:mysecretpassword@db:5432/postgres yarn knex -- migrate:latest
```

Comment:
Just spotted that we're actually setting the environment variable in the `docker-compose.yml` file [see here](https://github.com/ONSdigital/eq-author-api/blob/48e247e8cfc31bf1cb116fc2a6ed7c722c4f523f/docker-compose.yml#L16) so we can probably just change this line to `yarn knex -- migrate:latest`. Might want to give it a test though to be sure that works.